examples/output-function.ejs

```diff
@@ -0,0 +1,3 @@
+<p>
+  <strong><%= echo(name) %></strong> is a <%= echo(age) %> year old <% echo(species) %>.
```

Comment:
Looks like you neglected to change all tags to script-execution tags. In fact, you might want to make your example look more like your real use-case. For example, you might want to use output during debugging or something? You could even include a couple of examples of different ways to use it. It's good for people to know *why* this feature exists.